src/game_map.cpp

```diff
@@ -258,9 +258,26 @@ void Game_Map::SetupCommon(int _id) {
 	pan_wait = false;
 	pan_speed = 0;
 
-	// Make RPG_RT happy
-	// Otherwise current event not resumed after loading
-	location.map_save_count = map->save_count;
+	auto map_save_count = map->save_count;
+	if (Data::system.ldb_id == 2003 && map->save_count_2k3e > 0) {
```

Comment:
instead of ``Data::system.ldb_id == 2003`` use Player::IsRPG2k3

Reply:
Fixed